code/controllers/communications.dm

```diff
@@ -191,9 +191,6 @@ var/const/RADIO_MAGNETS = "radio_magnet"
 
 var/global/datum/controller/radio/radio_controller
 
-/hook/startup/proc/createRadioController()
```

Comment:
atomise the hook removal

Reply:
Welp now I regret not atomizing my commits :/ Sorry spooks